### ISSUE TITLE
master tries to import parse_options, which no longer exists

### ISSUE THREAD

**Issue by lvh** [GreenSteam/pep257#92]
```
pip install git+https://github.com/GreenSteam/pep257.git@8b331a850a45c57c43a762ff8133bcdea0bb3956#egg=pep257                                                                           09:21:05
Collecting git+https://github.com/GreenSteam/pep257.git@8b331a850a45c57c43a762ff8133bcdea0bb3956
  Cloning https://github.com/GreenSteam/pep257.git (to 8b331a850a45c57c43a762ff8133bcdea0bb3956) to /var/folders/2w/kffqc_pj6m38rgrxlhk8z6nr0000gn/T/pip-gRke1t-build
  Could not find a tag or branch '8b331a850a45c57c43a762ff8133bcdea0bb3956', assuming commit.
Installing collected packages: pep257
  Running setup.py install for pep257
    changing mode of build/scripts-2.7/pep257 from 644 to 755
    changing mode of /Users/lvh/.pyenv/versions/otter/bin/pep257 to 755
Successfully installed pep257-0.4.0
~/C/r/otter: pep257                                                                                                                                                                                 09:21:10
Traceback (most recent call last):
  File "/Users/lvh/.pyenv/versions/otter/bin/pep257", line 3, in <module>
    from pep257 import main, parse_options
ImportError: cannot import name parse_options
```

#75 might fix this accidentally (by replacing the current option parser). Cc @Nurdok?

**Comment by Nurdok** [comment]
It looks like I forgot to change this file while working on the config file support PR. Although the main failure here is that there are no tests for the script file. I will attempt to fix this today.

**Comment by Nurdok** [comment]
Solved by #93.
